### DISCUSSION TITLE
Converting whisper model to openvino fails

### DISCUSSION THREAD

**Discussion by sp00ktober** [openai/whisper#1697] [Q&A]
Morning, im trying to follow the steps from the Readme to convert the whisper model to openvino (to try it on my rPI4). Im doing this in a docker container on my Windows PC, installed the `openvino-conversion-requirements.txt`

Running `python convert-whisper-to-openvino.py --model large` with the `large` model produces the following error after a while:

```
100%|█████████████████████████████████████| 2.87G/2.87G [19:57<00:00, 2.58MiB/s]
/models/whisper.cpp/models/openvino_conv_env/lib/python3.10/site-packages/whisper/model.py:166: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  assert x.shape[1:] == self.positional_embedding.shape, "incorrect audio shape"
Killed
```

Is the large model not supported? Am i doing something wrong?
This also happens on the `medium` model.

**Comment by phineas-pta** [comment]
wrong repo, your question should be in https://github.com/ggerganov/whisper.cpp

**Reply by sp00ktober** [reply]
woops, you are right. Im sorry.
